bin/setup_new

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+# taken from https://gist.github.com/gvarela/9370838
+
+{
+  RUBY_VERSION=$(cat .ruby-version)
+  RUBY_VERSION=${RUBY_VERSION:=2.3.0}
+  PROJECT_NAME="ifme"
+} >/dev/null 2>&1
+
+## bootstrap function called at bottom of file comment out verification steps you don't need
+init (){
+  verify_ruby_version &&
+    verify_bundled_gems
+  verify_env_file
+  copy_example_config_files
+  verify_postgres_running &&
+    verify_database_migrations
+}
+
+
+## utility functions to test dependencies
+is_darwin (){
+  PLATFORM=$(uname)
+  [ "$PLATFORM" == "Darwin" ]
+}
+
+has_homebrew (){
+  [ is_darwin ] && [ hash brew >/dev/null 2>&1 ]
+}
+
+verify_homebrew (){
+  if [ ! has_homebrew ]
+  then
+    if [ is_darwin ]
+    then
+      echo 'Please install homebrew. Install instructions at http://brew.sh/'
+    else
+      echo "Please install Ruby version $RUBY_VERSION"
+    fi
+    exit 1
+  fi
+}
+
+has_postgres (){
+  hash psql >/dev/null 2>&1
```

Comment:
Ahh, got it. I was expecting output in addition to the exit code. Thanks for walking me through this! :D